src/layers/ngm-layers.ts

```diff
@@ -1,46 +1,38 @@
 import {html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
 import {repeat} from 'lit/directives/repeat.js';
-
 import {LitElementI18n} from '../i18n.js';
+import {Config} from './ngm-layers-item';
+// eslint-disable-next-line no-duplicate-imports
+import './ngm-layers-item';
```

Comment:
maybe better to move `Config` interface to `layertree.ts`

Reply:
Config is an extension of the `LayerTreeNode` Interface in `layertree.js` with some lit element respective ux specific config stuff. We could merge them but i would prefer not to to separate somewhat the data config from the ui/ux config.